### PR TITLE
Fix deployers

### PR DESCRIPTION
**Pull request by kichalla**
@Tratcher

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "projects": ["src"]
+    "projects": ["src","c:/github/testing/src","/home/kiran/github/testing/src"]
```
Comment (kichalla) [inline global.json]: will revert this change before checkin

File: src/Microsoft.AspNet.Server.Testing/Deployers/SelfHostDeployer.cs [hunk 2]
```diff
@@ -50,11 +51,11 @@ private CancellationToken StartSelfHost()
             var commandName = DeploymentParameters.Command;
             if (string.IsNullOrEmpty(commandName))
             {
-                commandName = DeploymentParameters.ServerType == ServerType.WebListener ? "web" : "kestrel";
+                commandName = DeploymentParameters.ServerType == ServerType.WebListener ? "weblistener" : "kestrel";
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Server.Testing/Deployers/SelfHostDeployer.cs]: FYI: The standard command names have changed to:
'web' = Kestrel
'weblistener = WebListener

**Comment by Tratcher** [comment]
:shipit:

**Comment by kichalla** [comment]
79a8a4e799a9ca78cd53ce19fcbba006f4c388ff
